src/Particle/Lattice/CrystalLattice.h

```diff
@@ -23,6 +23,7 @@
 #define OHMMS_CRYSTALLATTICE_H
 #include <limits>
 #include <iostream>
+#include <OhmmsData/Libxml2Doc.h>
```

Comment:
Is this really needed?

Reply:
I didn't mean using forward declare. I cannot find any use of xmlNodePtr under `src/Particle/Lattice` and thus feel the actual consumer of xmlNodePtr should have the include line.